docs/installation.md

```diff
@@ -0,0 +1,45 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Installing Federation v2](#installing-federation-v2)
+  - [Prerequisites](#prerequisites)
+    - [Creating Clusters](#creating-clusters)
+    - [Deployment Image](#deployment-image)
+  - [Helm Chart Deployment](#helm-chart-deployment)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Installing Federation v2
+
+## Prerequisites
```

Comment:
Please add reference to sourcing a kubefed2 binary from a recent release, as per #654.

Reply:
I think this should stay as a separate issue for now under #654 because it will require adding a generic kubefed2 installation doc somewhere else. 1. we would need details for this, and 2. I'm not sure where it would live if it's not specific to Federation only?

Reply:
I'm not sure what you mean by 'not specific to federation'.  `kubefed2` is very much specific to federation.  And the instructions are trivial:

 - download binary from the [desired release](https://github.com/kubernetes-sigs/federation-v2/releases/)
 - ensure the binary is in a directory referenced by the PATH argument

Reply:
This makes way more sense now we're renaming things - might make sense to add this in whatever PR we use to rename to kubefed in the docs?

Reply:
Instructions for `kubefedctl` installation have been added to the userguide since my original comment:

https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md#binaries